HeterogeneousTest/CUDAOpaque/test/testCUDATestAdditionModules.py

```diff
@@ -0,0 +1,32 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process('TestCUDATestOpaqueAdditionModule')
+
+process.source = cms.Source('EmptySource')
+
+process.CUDAService = cms.Service('CUDAService')
+
+process.cudaTestDeviceAdditionModule = cms.EDAnalyzer('CUDATestDeviceAdditionModule',
+    size = cms.uint32( 1024*1024 )
+)
+
+process.cudaTestKernelAdditionModule = cms.EDAnalyzer('CUDATestKernelAdditionModule',
+    size = cms.uint32( 1024*1024 )
+)
+
+process.cudaTestWrapperAdditionModule = cms.EDAnalyzer('CUDATestWrapperAdditionModule',
+    size = cms.uint32( 1024*1024 )
+)
+
+process.cudaTestOpaqueAdditionModule = cms.EDAnalyzer('CUDATestOpaqueAdditionModule',
+    size = cms.uint32( 1024*1024 )
+)
+
+process.path = cms.Path(
+    # this one fails with "cudaErrorInvalidDeviceFunction: invalid device function"
+    process.cudaTestDeviceAdditionModule +
```

Comment:
Following up on https://github.com/cms-sw/cmssw/pull/40637/files#r1091756724 on the ROCm side, this test should then fail on GPU_X IB, right?

Reply:
yes - ouch, I though I had commented it out :-(